client/verta/verta/_registry/modelversion.py

```diff
@@ -97,6 +98,13 @@ def _create_proto_internal(cls, conn, ctx, name, desc=None, tags=None, attrs=Non
         print("Created new ModelVersion: {}".format(model_version.version))
         return model_version
 
+    @classmethod
+    def from_run(cls, run_id):
```

Comment:
After some thought, maybe it should be `create_version_from_run` and be a method of `RegisteredModel`? I think we'll need the registered model info to attach, right?

Reply:
After some independent thought, I arrived at the same conclusion.